### PR TITLE
About page

### PR DESCRIPTION
**Pull request by Avuerro**
created separate layout file for the About page

### CODE REVIEW

**Review by v-bosch — CHANGES_REQUESTED** [review]
Looks great! In some cases I think it would be nice to write "AI" fully, as not everyone may know the abbreviation

File: _config.yml [hunk 1]
```diff
@@ -16,6 +16,14 @@ collections:
   tools:
     output: true
     permalink: /tools/:title/
+  team-members:
+    order:
+      - victoria.md
+      - eric.md
+      - akke.md
+      - marc.md
+      - chinari.md
```
Comment (a-t-0) [inline _config.yml]: I think we should rename `Chinari` to `Subhechha`.

File: about.md [hunk 2]
```diff
@@ -1,37 +1,12 @@
...
 title: About
-description: Learn all about the Markdown Guide.
+description: Learn about the origins of TruCol and the team behind TruCol.
 ---
 
-## Purpose
+## The origins of TruCol
 
-*The Markdown Guide* is a comprehensive Markdown reference designed for both novices and experts. It was born out of frustration with existing Markdown references that are incomplete, inadequate, or both.
+TruCol was founded as a project for the ETHDenver Hackathon as a collaboration between four students interested in the development of Ethereum projects. After competing in various categories, and successfully claiming multiple bounties, the team decided to continue with the development of the project. Other students who were interested in the project joined the team and several blockchain-related organisations were interested in the project.
```
Comment (a-t-0) [inline about.md]: We experienced that we did not always have the time or a skill for specific coding work ourselves when setting up our own company. Setting out bounties would provide a great solution for this. However, existing systems require an often costly middleperson and a modicum of trust between all parties. This is why we set out to develop a decentralized, trust-less collaboration platform and protocol for test-driven programming development for the ETHDenver Hackathon as a collaboration between four students interested in the development of Ethereum projects. After competing in various categories, and successfully claiming multiple bounties, the team decided to continue with the development of the project.

**Review by a-t-0 — CHANGES_REQUESTED** [review]

File: _config.yml (hunk 1 above)
Comment (Avuerro) [reply _config.yml]: done !

File: about.md (hunk 2 above)
Comment (Avuerro) [reply about.md]: I like your suggestion, except for one sentence ( see bold sentence below), I think it is too long.

_We experienced that we did not always have the time or a skill for specific coding work ourselves when setting up our own company. Setting out bounties would provide a great solution for this. However, existing systems require an often costly middleperson and a modicum of trust between all parties. ** This is why we set out to develop a decentralized, trust-less collaboration platform and protocol for test-driven programming development for the ETHDenver Hackathon as a collaboration between four students interested in the development of Ethereum projects **. After competing in various categories, and successfully claiming multiple bounties, the team decided to continue with the development of the project._

I would like to suggest this change, let me know what you think

_We experienced that we did not always have the time or a skill for specific coding work ourselves when setting up our own company. Setting out bounties would provide a great solution for this. However, existing systems require an often costly middleperson and a modicum of trust between all parties. **This is why we set out to develop a decentralized, trust-less collaboration platform and protocol for test-driven programming development. The project started at the ETHDenver Hackathon as a collaboration between four students interested in the development of Ethereum projects.** After competing in various categories, and successfully claiming multiple bounties, the team decided to continue with the development of the project._

**Comment by Avuerro** [comment]
> Looks great! In some cases I think it would be nice to write "AI" fully, as not everyone may know the abbreviation

I agree, I have changed it to Artificial Intelligence.

File: about.md (hunk 2 above)
Comment (a-t-0) [reply about.md]: Excellent!

**Pull request merged**